engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -4042,16 +4042,6 @@ private NicProfile orchestrateAddVmToNetwork(final VirtualMachine vm, final Netw
             //3) Convert nicProfile to NicTO
             final NicTO nicTO = toNicTO(nic, vmProfile.getVirtualMachine().getHypervisorType());
 
-            if (network != null) {
-                final Map<NetworkOffering.Detail, String> details = networkOfferingDetailsDao.getNtwkOffDetails(network.getNetworkOfferingId());
-                if (details != null) {
-                    details.putIfAbsent(NetworkOffering.Detail.PromiscuousMode, NetworkOrchestrationService.PromiscuousMode.value().toString());
-                    details.putIfAbsent(NetworkOffering.Detail.MacAddressChanges, NetworkOrchestrationService.MacAddressChanges.value().toString());
-                    details.putIfAbsent(NetworkOffering.Detail.ForgedTransmits, NetworkOrchestrationService.ForgedTransmits.value().toString());
-                }
-                nicTO.setDetails(details);
-            }
```

Comment:
@shwstppr are these details settings moved to Network Orchestration (when setting Nic profile there), right?

Reply:
@sureshanaparti moved to HypervisorGuruBase. The method at line 4043 was calling `HypervisorBaseGuru::toNicTO` and then setting these details. Since `HypervisorBaseGuru::toNicTO` has been refactored to include this code setting these details again was not needed